code/src/components/AddTask.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { useDispatch } from 'react-redux'
+
+import { tasks } from 'reducers/tasks'
+
+export const AddTask = () => {
+  const dispatch = useDispatch()
+  //State for input from textbox
+  const [value, setValue] = useState("")
+
+
+  //Handle submit function to dispatch task
+  const handleSubmit = (event) => {
+    event.preventDefault()
+
+    //Dispatch the action to save the new task.
+    dispatch(
```

Comment:
I noticed that there was a possibility of adding a task without any text in it. Maybe add some sort of conditional in here to prevent that empty tasks end up in the list. Like so:
<img width="449" alt="Skärmavbild 2020-11-30 kl  22 49 07" src="https://user-images.githubusercontent.com/47143197/100669944-70f55200-335e-11eb-95c9-f3bd58c8b057.png">

Reply:
Yup, this need to be fixed 👍

Reply:
Also, the design goes a little whacky after submitting many todos 🙈 
![Skärmavbild 2020-12-17 kl  18 44 12](https://user-images.githubusercontent.com/51529201/102523463-dbfd9300-4097-11eb-9b05-ab3ddef000f9.png)

Reply:
Great job with the changes Sara! Just this left to be fixed ☝️